### PR TITLE
Doc cleanup

### PR DESCRIPTION
**Pull request by LisaDelaney**

### CODE REVIEW

File: docs/sphinx/_toc.yml.in [hunk 1]
```diff
@@ -4,20 +4,47 @@ defaults:
...
-  - file: MI200AlternateImplementation
-  - file: MIOpen_Porting_Guide
-  - file: apireference
-- caption: About
+  - file: what-is-miopen.rst
+    title: What is MIOpen?
+
+- caption: Reference
+  entries:
+  - file: doxygen/html/index.html
```
Comment (samjwu) [inline docs/sphinx/_toc.yml.in]:
```suggestion
  - file: doxygen/html/index
```

Comment (LisaDelaney) [reply docs/sphinx/_toc.yml.in]: neither works...tested both

File: docs/index.rst [hunk 2]
```diff
@@ -1,16 +1,46 @@
...
+  .. grid-item-card:: Install
+
+    * :doc:`Install MIOpen <./install/install>`
+    * :doc:`Build MIOpen for embedded systems <./install/embed>`
+    * :doc:`Build MIOpen using Docker <./install/docker-build>`
+
+  ..  grid-item-card:: API reference
+
+    * :doc:`API library <./doxygen/html/modules>`
+      * :doc:`Modules <./doxygen/html/modules>`
```
Comment (yhuiYH) [inline docs/index.rst:27]:
```suggestion
    * :doc:`Modules <./doxygen/html/modules>`
```

Comment (LisaDelaney) [reply docs/index.rst:27]: needed a blank line there...

**Comment by LisaDelaney** [comment]
@pfultz2 Hi there, I think some double-checking may be required in the header file cross-references...a few of the includes seemed a little angry. Are you able to verify that the API bits are building with all the correct info?

**Review by randyh62 — APPROVED** [review]
Looks good to me.

**Comment by LisaDelaney** [comment]
@xinlipn @urpetkov-amd @pfultz2 are you able to review/approve this PR?

**Pull request merged**